src/lib/components/layout/FormLayout/FormLayout.jsx

```diff
@@ -1,8 +1,8 @@
-import flattenChildren from 'react-keyed-flatten-children';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { withProviderContext } from '../../../provider';
 import styles from './FormLayout.scss';
```

Comment:
It is small thing. However, you sometimes place `FormLayoutContext` before `styles`, sometimes the other way. Can you make it same for all occurances?

Reply:
I think ordering works like this:

* Imports are ordered from farthest away to the closes ones. (checked by eslint)
* Imports of the same distance are ordered alphabetically. (not checked by eslint)

It is correct here. If I placed it wrong somewhere I will fix it, but IMHO here is nothing to do. Are there any places in this PR where it is wrong?

Reply:
I also do it by import name (leftmost name) same as @adamkudrna 

I fixed this occurance.